test/java/seedu/addressbook/login/LoginTest.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.login;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.Parameterized;
+
+import static org.junit.Assert.*;
+
+import seedu.addressbook.login.Login;
+
+public class LoginTest {
+
+    private static final String SUCCESS = "Login Success, loading...";
+    private static final String Invalid_USERNAME_OR_PASSWORD = "Invalid username or password, please try again";
+    private static final String Invalid_FORMAT = "Please input the correct login command: login username password";
+    private Login login = new Login();
+
+    @Test
+    public void setLoginStatus() {
+        login.setLoginStatus(true);
+        assertEquals(login.getLoginStatus(), true);
+    }
+
+    @Test
+    public void getLoginStatus() {
+        login.setLoginStatus(false);
+        assertEquals(login.getLoginStatus(), false);
+    }
+
+    @Test
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.